Samples/Xbox/InteractivitySample.cpp

```diff
@@ -621,6 +648,11 @@ void Sample::ProcessInteractiveEvents(std::vector<interactive_event> events)
         case interactive_event_type::error:
             HandleInteractivityError(event);
             break;
+        case interactive_event_type::unknown:
+            std::shared_ptr<interactive_message_event_args> messageEventArgs = std::dynamic_pointer_cast<interactive_message_event_args>(event.event_args());
+            m_console->Write(messageEventArgs->message().c_str());
+            m_console->Write(L"\n");

```

Comment:
(hint - you can reply to a comment directly by hitting that curved little arrow)

---
In reply to: [132342774](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132342774) [](ancestors = 132342774)